xilem_web/README.md

```diff
@@ -1,9 +1,32 @@
-# `xilem_web` prototype
+# `xilem_web`
```

Comment:
We could plausibly follow the usual readme patterns here, but that probably isn't a job for this pr

Reply:
You're right, I've copied a lot of the Xilem Core Readme and actually used the same pattern to include it in the `lib.rs` now, you probably want to look quickly at it again before I merge.

Reply:
I'll update both once this is merged. Thanks!